### DISCUSSION TITLE
Tooltip over box annotation

### DISCUSSION THREAD

**Discussion by jirikolarik** [chartjs/chartjs-plugin-annotation#908] [Q&A]
Hey, is it possible to show tooltip on hover over box annotation?

**Comment by stockiNail** [comment]
@jirikolarik not yet. Anyway the PR #892 could enable it, when be applied, maybe developing a custom interaction

**Comment by tomasbeloch** [comment]
@stockiNail that would be great if we could push this one forward, I would also appreciate this feature.
